quotefaultAPI/__init__.py

```diff
@@ -208,17 +166,9 @@ def newest(api_key: str):
     :return: Returns the newest quote found during the query
     """
     if check_key(api_key):
-        date = request.args.get('date')
+		date = request.args.get('date')
```

Comment:
Please don't use tab characters. The rest of the application uses 4 spaces as is PEP8